Source/Embeddings.Store/ProjectionKeyToEventSourceIdConverter.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Data.HashFunction;
+using System.Data.HashFunction.CityHash;
+using System.Text;
+using Dolittle.Runtime.Events.Store;
+using Dolittle.Runtime.Projections.Store;
+
+namespace Dolittle.Runtime.Embeddings.Store
+{
+    /// <summary>
+    /// Represents an implementation of <see cref="IConvertProjectionKeysToEventSourceIds"/>.
+    /// </summary>
+    public class ProjectionKeyToEventSourceIdConverter : IConvertProjectionKeysToEventSourceIds
+    {
+        readonly Encoding _encoding;
+        readonly ICityHash _hasher;
+
+        public ProjectionKeyToEventSourceIdConverter()
+        {
+            _encoding = Encoding.Unicode;
+            _hasher = CityHashFactory.Instance.Create(new CityHashConfig { HashSizeInBits = 128 });
```

Comment:
This 128 number should be a variable, it is used two places. I suggest it to be a const int

Reply:
Like what I did now? Codacy doesn't like consts apparently, it wants a static readonly property. What do we want?

Reply:
I would prefer const over static readonly in this scenario as it resolves at compile-time rather than a runtime lookup. static readonly is good for initializing constant values that does not have a compile-time value. I think we'll look at tweeking some sonarqube rules at some point for codacy

Reply:
Funny, I found this https://rules.sonarsource.com/csharp/tag/performance/RSPEC-3962

Reply:
How about now?

Reply:
I like